recipes/mosaic-clustering/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set name = "mosaic-clustering" %}
+{% set version = "0.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/mosaic-clustering-{{ version }}.tar.gz
+  sha256: aa6b3342c517cf4f0bcf57f9d80e80108d54490cab6e7bba13e38ab28f0af4ce
+
+build:
+  skip: true  # [py<38]
```

This could be made `noarch: python` by removing the selector from `typing-extensions` (which might not be needed but is available for all Python versions and then you get the various noarch perks):
```suggestion
  noarch: python
```

---

Thx, for pointing this out. I moved to noarch by making `typing-extensions` an explicit dependency and requiring a lower python version.